src/components/Modal/Modal.jsx

```diff
@@ -64,6 +64,13 @@ class Modal extends React.Component {
     }
   };
 
+  validateAction = action => {
```

Comment:
I moved this method up because I was getting an error locally and couldn't load the documentation.